### PR TITLE
Create ginny-caughey-registration.md

### PR DESCRIPTION
**Pull request by gcaughey**
Speaker list registration

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=540) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=540) before we can accept your contribution.<br/><br/>:x: gcaughey [sign now](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=540)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-foundation/website?pullRequest=540) it.</sub>

**Comment by clairernovotny** [comment]
The twitter and github links need to be full URL's.

File: input/community/speakers/ginny-caughey-registration.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Language:
+  - English
+Topics:
+  - .NET
+  - C#
+  - UWP
+  - Windows Development
+  - Windows Forms
+  - WPF
+Twitter: @gcaughey
```
Comment (gcaughey) [inline input/community/speakers/ginny-caughey-registration.md:14]: https://twitter.com/gcaughey

File: input/community/speakers/ginny-caughey-registration.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+GitHub: gcaughey
```
Comment (gcaughey) [inline input/community/speakers/ginny-caughey-registration.md:15]: https://github.com/gcaughey

File: input/community/speakers/ginny-caughey-registration.md (hunk 1 above)
Comment (gcaughey) [inline input/community/speakers/ginny-caughey-registration.md:14]: https://twitter.com/gcaughey

File: input/community/speakers/ginny-caughey-registration.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+Mentor: true
+---
+For several decades I've been a speaker at big tech conferences such as Build, and TechEd, tiny meetings like my local .NET user group, and many in-between sized conferences in the US, Europe, UK, and Australia.  I've even participated in several on-line events as a speaker, so I'm very familiar with stage fright and imposter syndrome in all ifs forms. ;-) But I still enjoy sharing what I've learned with others.
```
Comment (gcaughey) [inline input/community/speakers/ginny-caughey-registration.md:18]: For several decades I've been a speaker at big tech conferences such as Build, and TechEd, tiny meetings like my local .NET user group, and many in-between sized conferences in the US, Europe, UK, and Australia.  I've even participated in several on-line events as a speaker, so I'm very familiar with stage fright and imposter syndrome in all iys forms. ;-) But I still enjoy sharing what I've learned with others.

**Pull request merged**